sdk/python/lib/pulumi/output.py

```diff
@@ -297,6 +297,35 @@ def from_input(val: Input[T_co]) -> 'Output[T_co]':
         value_fut.set_result(val)
         return Output(set(), value_fut, is_known_fut, is_secret_fut)
 
+    @staticmethod
+    def _from_input_shallow(val: Input[T]) -> 'Output[T]':
+        """
+        Like `from_output`, but does not recur deeply. Instead, checks if `val` is an `Output` value
```

Comment:
Did you mean:

```suggestion
        Like `from_input`, but does not recur deeply. Instead, checks if `val` is an `Output` value
```

?